chat-plugins/trivia.js

```diff
@@ -686,13 +686,13 @@ var commands = {
 
 			if (isAccepting) {
 				var accepted = [];
-				for (var i = indicesLen; i--;) {
+				for (var i = 0; i < indicesLen; i++) {
```

Comment:
Looping forward means you need to use offsets on indices (which is what I usually do, but then again I usually only have a single index rather than an array of them for which using offsets gets much more complicated).